python/aikidopy/CMakeLists.txt

```diff
@@ -94,6 +88,12 @@ set_target_properties(aikidopy
     DEBUG_POSTFIX ""
 )
 
+execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
```

Comment:
Ditto: What's the reason for this change? This may be dumb, seems unrelated to the interface change.

Reply:
Just a small readability change, since this variable is only used in the below lines.